src/activation.rs

```diff
@@ -25,6 +26,12 @@ pub trait IsType {
     /// Returns true if a file descriptor is a `PF_UNIX` socket.
     fn is_unix(&self) -> bool;
 
+    /// Returns true if a file descriptor is a `SOCK_STREAM` socket.
+    fn is_stream(&self) -> bool;
```

Comment:
Just asking: is this (and the one below) only supposed to cover UNIX sockets or others (TCP, UDP, etc) too, in the future? It may be good to make that clear in the docstrings here.